lib/core/mount.js

```diff
@@ -168,9 +169,13 @@ function mount(mountPath, parentApp, events) {
 
 		// Initialize the session store
 		try {
-			
-			var SessionStore = require(sessionStore)(express);
-			
+      if(sessionStore === 'connect-redis') {
+				expressSession = require('express-session');
```

Comment:
It's available in master, but not in the 2.x branch. Are you building future 2.x versions from the 2.x branch or from master?